src/Dataporten/Provider.php

```diff
@@ -57,4 +62,17 @@ protected function getTokenFields($code)
             'grant_type' => 'authorization_code',
         ]);
     }
+
+    /**
+     *  Get the endsession endpoint URL.
+     */
+    public function getLogoutUrl($idToken, $endpointUri, $redirectUri)
+    {
+        $params = http_build_query(array_filter([
```

Comment:
This method doesn't really do anything. Could we put some of these field as config, and the have the provider build it automatically?

Reply:
Could we also get strict types please